libs/platform/src/lib/wizard-generator/components/wizard-generator-step/wizard-generator-step.component.ts

```diff
@@ -197,18 +197,15 @@ export class WizardGeneratorStepComponent implements OnInit, OnDestroy, OnChange
     }
 
     /**
-     * @returns Step submitted forms values.
+     * Returns visible forms in the step.
      */
-    getFormValues(): WizardStepSubmittedForms {
```

Comment:
is this a breaking change?

Reply:
wouldn't say so, although it's exposed we don't use `fdp-wizard-generator-step` anywhere